src/tests/Common/Directory.Build.targets

```diff
@@ -122,7 +137,7 @@
             $(Crossgen2Dir)/clrjit_*;
             $(Crossgen2Dir)/jitinterface_*;
             $(Crossgen2Dir)/Microsoft.DiaSymReader.Native.*.dll"
-          TargetDir="crossgen2/" />
+          TargetDir="../crossgen2/" />
```

Comment:
Can you please explain the reason for this change? According to what I see at line 305, it seems like the double period would crawl out of <code>CORE_ROOT</code>.

Reply:
This is producing the crossgen2 directory out of `CORE_ROOT`, then in `src/tests/build.cmd/sh` we `move` it back inside the `CORE_ROOT`. There was a problem with MSBuild mixing the files to copy selected by glob without this workaround (and I actually don't remember whether this workaround was enough for all publishing modes, will try to re-run the tests when I get some free cycles).

Reply:
Hmm, that sounds weird, I would rather suspect that what's actually happening is that for some reason the population of CORE_ROOT tries to copy both versions of crossgen2 and that naturally fails due to clashing duplicates. If you have trouble double-checking that in binlogs, I can take a look locally and try to figure out what's going on, we shouldn't need to copy the files around twice as you describe.

Reply:
It is done on line 41 via `PublishDir=$(CORE_ROOT)\..\crossgen2`. This part is in sync with that path. IIRC, the issue was with the list generated by MSBuild via glob misses some files. I was comparing the text logs generated via `-v:diag`. Will run again to refresh the memory.